tests/test_collection.py

```diff
@@ -120,7 +120,7 @@ def test_insert(self):
         assert self.c["0"] == doc
         assert list(self.c.find()) == [doc]
         with pytest.raises(ValueError):
-            self.c["0"] = dict(_id="1")
+            self.c["0"] = dict(id_="1")
```

Comment:
Let's revert the changes in this file. We are going to remove the `Collection` class and its tests soon, but these tests probably rely on `_id` being the primary key defined here: https://github.com/glotzerlab/signac/blob/c2a481ef5f144bad27b2a856c267a63294bff99b/signac/contrib/collection.py#L420

Reply:
Fantastic - that's all been sorted